homeassistant/helpers/config_validation.py

```diff
@@ -233,8 +233,14 @@ def template(value):
     """Validate a jinja2 template."""
     if value is None:
         raise vol.Invalid('template value is None')
-    if isinstance(value, (list, dict)):
-        raise vol.Invalid('template value should be a string')
+    if isinstance(value, list):
```

Comment:
I think this variant is more flexible and don't break anythings and reduce complex logic